### ISSUE TITLE
accessibility issue: o-table striped has low color contrast for links inside the darker rows

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#486]
When using o-table with stripes and a table cell in a darker row contains a link, the color contrast of the link compared to the background of the row is too low and does not meet the WCAG standards for accessible text.

The contrast ratio is currently 4.14 - we need to reach at least 4.5 to meet the WCAG standards.

![Screenshot of a striped table containing a link, an accessibility tooltip shows the contrast ratio is 4.14 which is too low](https://user-images.githubusercontent.com/1569131/147086270-a2ea8431-5644-41fd-8027-c3dc90c1ad4d.png)

**Comment by JakeChampion** [comment]
The current background color for the stripes is wheat
https://github.com/Financial-Times/origami/blob/c34658ed33199052710d99ba8010f626b62fda97/components/o-table/src/scss/_brand.scss#L17

I reckon we will need to decide on a different background color here instead of wheat.

I've posted in the design-system-guild channel -- https://financialtimes.slack.com/archives/C01481FKWA2/p1640173461118500

**Comment by notlee** [comment]
https://github.com/Financial-Times/origami/pull/771
